spec/models/ability/team_member_spec.rb

```diff
@@ -61,16 +61,12 @@
       end
 
       context "when viewing user information" do
-
-        # FIXME see issue #1001
-        # The following specs should pass after fixing
         before do
           team_member.update!(hide_email: true)
           user_in_other_team.update!(hide_email: true)
         end
 
         it "can read email address of team members in current season, even when hidden" do
-          pending 'fails; To be solved with issue #1001'
           allow(subject).to receive(:supervises?).with(team_member, user).and_return true
           expect(subject).to be_able_to(:read, team_member.email)
```

Comment:
I'm not entirely sure how this test was originally intended... But from what I remember, `CanCan` works on objects only, it's not fine-grained enough for attributes 😉 

So this should rather be:
```ruby
expect(subject).to be_able_to(:read_email, team_member)
```

And while we're at it, I think the spec would be easier to read and maintain if we'd use the actual subjet's name, like this:

```ruby
expect(ability).to be_able_to(:read_email, team_member)
```

Or, use the shorthand syntax:

```ruby
before { allow(ability).to receive(:supervises?).with(team_member, user).and_return(true) }

it { is_expected.to be_able_to(:read_email, team_member) }
```